evap/static/css/evap.scss

```diff
@@ -44,6 +45,26 @@ div.container {
     padding-top: 80px; // for navbar (and breadcrumbs)
     flex: 1 0 auto;
 }
+.sided-center {
+    display: flex;
+    > :first-child {
+        flex: 1;
+    }
+    > :nth-child(2) {
+        flex: 5;
+        text-align: center;
+    }
+    > :last-child {
+        flex: 1;
+        text-align: right;
+    }
+}
```

Comment:
what does this do?

Reply:
it makes an element with three children placing the first with left-aligned content, the last with right-aligned content and the middle one (the second) with center-aligned content - this being five times as large as the outer ones. it's used for the quick review's card footer (and i think also for the page footer, but i didn't review the last changes yet).

Reply:
page footer and navigation bar was identified as further applications of this style class, was also tested on my local machine, but should be in a separate PR.
Also, I am open for better names. My alternatives: `three-splitted-row` `left-center-right` `centered-row`